kornia/augmentation/_2d/geometric/base.py

```diff
@@ -167,7 +167,7 @@ def inverse_masks(
         transform: Optional[Tensor] = None,
         **kwargs,
     ) -> Tensor:
-        resample_method: Optional[Resample]
+        resample_method: Optional[Resample] = None
```

Comment:
This fixed the issue you told me about?  :flushed:

Reply:
No, I wish! Without this, an error is thrown straight away and I didn't even tell you about it yesterday. The other issue appears with this patched.